contents/blog/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,121 @@
+
+I went to a superb talk by Jeff Lawson (who runs Twilio), who felt the greatest innovation at Amazon was how it felt like thousands of small startups. Simply speaking, a startup gets more done per person than a big corporate.
+
+PostHog works like a group of startups. For each of our products, there is a team of up to 6, who can ship with minimal interference from the rest of the company. We hire a lot of ex technical founders to help here.
+
+## Nearly everyone is an engineer
+
+It's simple, but if you have lots of spending and focus outside of engineering, you won't get as much built.
+
+We are around 45 people, 38 of whom are engineers.
+
+We don't do outbound sales, our marketing team is very small, we only have one product manager, our exec team only has 3 people. These things are all by design (and were possible because of our market - the majority of engineers generally want docs and to try something instead of demos and salespeople to deal with).
+
+## No meeting Tuesdays/Thursdays
+
+We want a culture where people can deep work. We push back on unnecessary meetings (and yes, there are _definitely_ some that are necessary, we have learned - often painfully).
+
+## We are a late mover
+
+I've had lots of founders tell me that they can't pursue idea X because some other startups are tackling it. This makes me laugh because we simply won't build a new product unless there is a $1bn+ competitor already offering it. We _love_ competition. It shows you that there will be demand. The magic in our platform (after an individual product is working well for our users) is then integrating everything together.
+
+For non-God tier product people, such as me, I think it's easier to build a better version of X than it is to create something entirely new.
+
+## We raised venture capital
+
+Obviously, this matters. We didn't raise or spend a ridiculous amount compared to other companies we compete with - we have done a $3M seed, $9M series A and a $15M series B. We have a lot of it left and will be profitable next year.
+
+Raising money does let build more stuff, but if you _should_ is very questionable and a topic for another time.
+
+## Trust and feedback over process
+
+This is one of our values. It's simply up to the person building in most situations. Building and scaling something people want is a nuanced problem, so we let people use their judgement. When they get it wrong, we are direct and give feedback.
+
+To quote one of our team "process is scar tissue" - it often overcorrects, but because humans are risk adverse it feels natural to implement them when a company starts growing 
+
+## There are plenty of problems
+
+As a company with high autonomy and low coordination, shipping things that affect multiple teams is _a lot_ harder than shipping something within a single team. You may need extra meetings, or to form a temporary project team, or you may just fail!
+
+At scale, this has meant performance has, well, um, been a performance to solve.
```

Comment:
```suggestion
Since we handle a lot of data, at scale, this has meant performance has, well, um, been a performance to solve. Performance is a very broad area to solve for - ranging from our infrastructure all the way through to our UX, and thus involves a lot of coordination and dependencies.
```